README.md

```diff
@@ -134,24 +134,39 @@ Usage:
 - `abc templates golden-test record [--test-name=<test_name>] [<location>]`
 - `abc templates golden-test verify [--test-name=<test_name>] [<location>]`
 
+Note: for `new-test`, the `<location>` parameter gives the location of the template.
```

Comment:
Could we explain that the location is a directory that will be recursively searched for templates and tests?